Tribler/Core/Modules/restapi/channels_endpoint.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from twisted.web import resource
+from twisted.web.resource import NoResource
+
+from Tribler.Core.simpledefs import NTFY_CHANNELCAST
+
+
+class BaseChannelsEndpoint(resource.Resource):
+    """
+    This class contains some utility methods to work with raw channels from the database.
+    All endpoints that are using the database, should derive from this class.
+    """
+
+    def convert_db_channel_to_json(self, channel):
+        return {"id": channel[0], "name": channel[2], "description": channel[3], "votes": channel[5],
```

Comment:
I made a mistake, it was the Dispersy community ID. Nevertheless, I've added it to the response, see my last fixup commit.